src/app/progress/milestone.scss

```
@@ -64,6 +65,7 @@
     line-height: 1.3;
     max-height: 3.9em;
     overflow: hidden;
+    word-break: break-word;
```

I never knew until I looked this up, but apparently this has been deprecated in favor of:
```suggestion
    overflow-wrap: break-word;
```

https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap

We should probably set that at the root so it applies everywhere.